src/Pages/Appointment.jsx

```diff
@@ -1,12 +1,19 @@
 import Navbar from "../components/Navbar";
 import Header from "../components/Header";
 import NewAppointment from "../components/NewAppointment";
+import { motion } from "framer-motion";
 
 function Appointment() {
   return (
     <>
       <Header titleText={"Termine"} />
-      <NewAppointment></NewAppointment>
+      <motion.div
+        exit={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        initial={{ opacity: 0 }}
+      >
+        <NewAppointment></NewAppointment>
```

Comment:
This could be a self-closing tag:
```suggestion
        <NewAppointment />
```